src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_defaults.py

```diff
@@ -299,9 +322,33 @@ def test_linux_with_password(self):
 
         # throw when conflict with ssh key value
         ns.ssh_key_value = 'junk but does not matter'
-        with self.assertRaises(ValueError) as context:
+        with self.assertRaises(CLIError) as context:
             _validate_vm_vmss_create_auth(ns)
-        self.assertTrue("incorrect usage for authentication-type 'password':" in str(context.exception))
+            self.assertTrue("SSH key cannot be used with password authentication type." in str(context.exception))
+
+    def test_linux_with_password_and_ssh_explicit(self):
+        ns = TestVMDefaultAuthType._set_ns()
+        ns.os_type = "LINux"
+        ns.authentication_type = 'all'
+        ns.admin_username = 'user12345'
+        ns.admin_password = 'verySecret!!!'
+        ns.ssh_key_value = self._get_test_ssh_key()
+
+        _validate_vm_vmss_create_auth(ns)
+        self.assertEqual(ns.authentication_type, 'all')
+        self.assertEqual(ns.ssh_dest_key_path, '/home/{}/.ssh/authorized_keys'.format(ns.admin_username))
+
+    def test_linux_with_password_and_ssh_explicit(self):
```

Comment:
this is a dupe test :)

Reply:
Hello Yugang, I will rename the test. The only difference is that the second test sets authentication_type to none